src/Menu.java

```diff
@@ -0,0 +1,156 @@
+import org.json.simple.JSONArray;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Menu {
+
+
+    public static void main(String[] args){
+
+        try {
+            int userInput;
+            do {
+                Utils.clearScreen();
```

Comment:
It seems `Util.clearScreen()` is being called a lot (8 times) and some more calls are still needed. It might be a good idea to make it the default behavior after the user enters any input, but also add a boolean argument to skip this behavior if needed. You could overload the `Keyboard.nextLine()` method to do this.